### ISSUE TITLE
Cannot configure the 2023.12.6

### ISSUE THREAD

**Issue by Djavdeteg** [ankohanse/hass-dab-pumps#8]
Hi.

After update from 12.5 all data became 'Unavailable' with:
`[custom_components.dabpumps.dabpumpsapi] Please re-check your username and password in your configuration!`

Ok, I removed 12.5 and did a fresh installation of 12.6.
I cannot complete configuration steps now because the last step asks me to choose installation name  without any opton to select.
After the last window closed with X, integration remains not installed.
`2024-01-01 19:43:52.287 INFO (MainThread) [custom_components.dabpumps.config_flow] Trying connection...
2024-01-01 19:43:58.447 WARNING (MainThread) [custom_components.dabpumps.dabpumpsapi] Unable to connect to dconnect.dabpumps.com.
2024-01-01 19:43:58.447 INFO (MainThread) [custom_components.dabpumps.config_flow] Successfully connected!`

![chrome_JA6mc15iNh](https://github.com/ankohanse/hass-dab-pumps/assets/111967435/182e5d92-4b64-4c88-81c2-62004223b74d)
![chrome_k6PP1zBELJ](https://github.com/ankohanse/hass-dab-pumps/assets/111967435/2f357c27-f1fe-4e0e-9a37-d3a445cd92c5)
![oBigVbI7kt](https://github.com/ankohanse/hass-dab-pumps/assets/111967435/95716841-7424-451a-8051-e577c206c1f7)

D.

**Comment by ankohanse** [comment]
Ran into the same issues while testing, but after a couple of retries I got through it. It is a strange issue, because no code changes were introduced in that part.
I strongly suspect that the problem is mainly with the DConnect.DabPumps.com website at the moment; when I try opening it in a web browser I often get an 'Internal Server Error' page displayed.

Will keep this issue report open for now, and will try to make my code more robust so that setup will not continue when no installation was found.
